<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Custom_Game_1</class>
 <widget class="QWidget" name="Custom_Game_1">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>673</width>
    <height>469</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_7">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="label">
     <property name="minimumSize">
      <size>
       <width>641</width>
       <height>41</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>24</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Versus Mode</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QGroupBox" name="GrpGameOpts">
       <property name="maximumSize">
        <size>
         <width>202</width>
         <height>392</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="title">
        <string>Game Options</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <property name="leftMargin">
         <number>9</number>
        </property>
        <property name="topMargin">
         <number>6</number>
        </property>
        <property name="rightMargin">
         <number>9</number>
        </property>
        <item>
         <widget class="QGroupBox" name="BoardOpts">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="title">
           <string>Board Selection</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QRadioButton" name="Board_8x8">
             <property name="text">
              <string>Chess [ 8x8 ]</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="Board_9x9">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>Shogi [ 9x9 ]</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="e_promodrop">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Promoted Drop</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="e_menpassant">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Metal Piece
en-passant</string>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="e_mdoublemove">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Metal Piece
Double initial move</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="CaptureOpts">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="title">
           <string>Capture Mode</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout">
           <property name="rightMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QRadioButton" name="c_crazyhouse">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>Crazyhouse</string>
             </property>
             <property name="checked">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="c_graveyard">
             <property name="text">
              <string>Graveyard</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="c_shogionly">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>Shogi Pieces Only</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="verticalFrame">
       <property name="maximumSize">
        <size>
         <width>253</width>
         <height>392</height>
        </size>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QGroupBox" name="GrpPlayerOpts">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>251</width>
            <height>390</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="title">
           <string>Player Options</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <widget class="QGroupBox" name="P1Side">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>58</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="title">
              <string>Player 1 Side</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
             <property name="flat">
              <bool>false</bool>
             </property>
             <widget class="QRadioButton" name="P1_Chess">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="geometry">
               <rect>
                <x>10</x>
                <y>30</y>
                <width>71</width>
                <height>19</height>
               </rect>
              </property>
              <property name="text">
               <string>Chess</string>
              </property>
              <property name="checked">
               <bool>false</bool>
              </property>
             </widget>
             <widget class="QRadioButton" name="P1_Shogi">
              <property name="geometry">
               <rect>
                <x>130</x>
                <y>30</y>
                <width>61</width>
                <height>19</height>
               </rect>
              </property>
              <property name="text">
               <string>Shogi</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="Lineup">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>151</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="title">
              <string>Lineup</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
             <property name="flat">
              <bool>false</bool>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <property name="spacing">
               <number>0</number>
              </property>
              <property name="leftMargin">
               <number>0</number>
              </property>
              <property name="rightMargin">
               <number>0</number>
              </property>
              <item>
               <widget class="QGroupBox" name="L_Shogi">
                <property name="minimumSize">
                 <size>
                  <width>101</width>
                  <height>121</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <pointsize>12</pointsize>
                 </font>
                </property>
                <property name="title">
                 <string>Shogi</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
                <property name="flat">
                 <bool>false</bool>
                </property>
                <widget class="QRadioButton" name="SL_Default">
                 <property name="geometry">
                  <rect>
                   <x>10</x>
                   <y>30</y>
                   <width>81</width>
                   <height>19</height>
                  </rect>
                 </property>
                 <property name="text">
                  <string>Default</string>
                 </property>
                 <property name="checked">
                  <bool>true</bool>
                 </property>
                </widget>
                <widget class="QRadioButton" name="SL_Mirror">
                 <property name="geometry">
                  <rect>
                   <x>10</x>
                   <y>60</y>
                   <width>71</width>
                   <height>19</height>
                  </rect>
                 </property>
                 <property name="text">
                  <string>Mirror</string>
                 </property>
                </widget>
                <widget class="QRadioButton" name="SL_Outpost">
                 <property name="enabled">
                  <bool>false</bool>
                 </property>
                 <property name="geometry">
                  <rect>
                   <x>10</x>
                   <y>90</y>
                   <width>81</width>
                   <height>19</height>
                  </rect>
                 </property>
                 <property name="text">
                  <string>Outpost</string>
                 </property>
                </widget>
               </widget>
              </item>
              <item>
               <widget class="QGroupBox" name="L_Chess">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>131</width>
                  <height>121</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <pointsize>12</pointsize>
                 </font>
                </property>
                <property name="title">
                 <string> Chess</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
                <property name="flat">
                 <bool>false</bool>
                </property>
                <widget class="QRadioButton" name="CL_Default">
                 <property name="geometry">
                  <rect>
                   <x>10</x>
                   <y>30</y>
                   <width>121</width>
                   <height>19</height>
                  </rect>
                 </property>
                 <property name="text">
                  <string>Default</string>
                 </property>
                 <property name="checked">
                  <bool>true</bool>
                 </property>
                </widget>
                <widget class="QRadioButton" name="CL_QDown">
                 <property name="geometry">
                  <rect>
                   <x>10</x>
                   <y>60</y>
                   <width>121</width>
                   <height>19</height>
                  </rect>
                 </property>
                 <property name="text">
                  <string>Queen Down</string>
                 </property>
                </widget>
                <widget class="QRadioButton" name="CL_RKnight">
                 <property name="geometry">
                  <rect>
                   <x>10</x>
                   <y>90</y>
                   <width>121</width>
                   <height>19</height>
                  </rect>
                 </property>
                 <property name="text">
                  <string>Royal Knight</string>
                 </property>
                </widget>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="GrpTimerOpts">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>130</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="title">
              <string>Timer</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
             </property>
             <property name="flat">
              <bool>false</bool>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
             <layout class="QGridLayout" name="gridLayout">
              <property name="leftMargin">
               <number>9</number>
              </property>
              <item row="4" column="0">
               <widget class="QCheckBox" name="checkByoyomi">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="layoutDirection">
                 <enum>Qt::LeftToRight</enum>
                </property>
                <property name="text">
                 <string>Byoyomi</string>
                </property>
                <property name="autoRepeat">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
              <item row="2" column="0">
               <widget class="QCheckBox" name="checkIncrement">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="text">
                 <string>Increment</string>
                </property>
               </widget>
              </item>
              <item row="3" column="1">
               <widget class="QTimeEdit" name="timeDelay">
                <property name="frame">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item row="2" column="1">
               <widget class="QTimeEdit" name="timeIncrement">
                <property name="frame">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item row="3" column="0">
               <widget class="QCheckBox" name="checkDelay">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="text">
                 <string>Delay</string>
                </property>
               </widget>
              </item>
              <item row="4" column="1">
               <widget class="QTimeEdit" name="timeByoyomi">
                <property name="frame">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QTimeEdit" name="timeTimer">
                <property name="displayFormat">
                 <string>mm:ss</string>
                </property>
                <property name="time">
                 <time>
                  <hour>0</hour>
                  <minute>10</minute>
                  <second>0</second>
                 </time>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="label_2">
                <property name="text">
                 <string>Duration</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="verticalFrame_2">
       <property name="maximumSize">
        <size>
         <width>213</width>
         <height>392</height>
        </size>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <widget class="QGroupBox" name="groupBox_5">
          <property name="minimumSize">
           <size>
            <width>211</width>
            <height>251</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="title">
           <string>Lineup Preview</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <widget class="QLabel" name="lineup_preview">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>40</y>
             <width>191</width>
             <height>191</height>
            </rect>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap>../resources/board_minimal.png</pixmap>
           </property>
           <property name="scaledContents">
            <bool>true</bool>
           </property>
          </widget>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="buttonAdvOpts">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>201</width>
            <height>31</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Advanced Options</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="buttonStart">
          <property name="minimumSize">
           <size>
            <width>201</width>
            <height>91</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>18</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Start Playing</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
